Source/Atomic/Scene/Node.h

```diff
@@ -602,6 +602,39 @@ class ATOMIC_API Node : public Animatable
     /// Return child scene nodes by name hash, optionally recursive
     void GetChildrenWithName(PODVector<Node*>& dest, StringHash nameHash, bool recursive = false) const;
 
+    /// Scripting interface to set user data, the data type can be 
+    ///   Int, Bool, Float, Vector2, Vector3, Vector4, Quaternion
+    ///   String, Buffer, ResourceRef, ResourceRefList, IntRect,
+    ///   IntVector2, Matrix3,Matrix3x4, Matrix4, Double, Color
+    /// The data value must be programed in the appropriate string form
+    /// used by the Variant class.
+    void setUserData (const String& name, const String& vartype, const String& value) 
+    { 

```

Comment:
C++ method naming convention is camel case capitals, so for example ```SetUserData```

We're mixing ```SetVar``` and ```SetUserData``` when referring to the same ```var_```  VariantMap member.  Something like```SetVarFromString``` would be consistent with ```Var``` and ```Variant::FromString```

On the comment, how about, "The data value must be formatted in the Variant type string format"?